backend/routes/menuInfo.js

```diff
@@ -319,6 +324,8 @@ router.get("/prefsAndRests", async (req, res) => {
 
 // this endpoint returns all menu items of the provided dining court
 router.get("/:diningCourt", async (req, res) => {
+    var d = new Date();
+    var today = new Date(d.getFullYear() + "-" + (d.getMonth() + 1) + "-" + d.getDate());

```

Comment:
and removed `var today = new Date(d.getFullYear() + "-" + (d.getMonth() + 1) + "-" + d.getDate());`